scripts/fetchHarvesterTypes.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable */
+const git = require('nodegit');
+const path = require('path');
+const rimraf = require('rimraf');
+const { exec } = require('child_process');
+
+const BRANCH = process.env.BRANCH;
+const GITHUB_PROYECT = 'git@github.com:PolymathNetwork/tooling-gql.git';
+const TOOLING_FOLDER = 'tooling-gql';
+const OUTPUT_TYPES = path.resolve(__dirname, '../src/polkadot/harvester-types.ts');
```

Comment:
This has nothing to do with polkadot, it should go into its own folder

`src/harvester/types.ts`